gap/lex.g

```diff
@@ -1,272 +0,0 @@
-###########################################################################
-##
-#W  lex.g               OpenMath Package                     Andrew Solomon
-#W                                                         Marco Costantini
-##
-#Y    Copyright (C) 1999, 2000, 2001, 2006
-#Y    School Math and Comp. Sci., University of St.  Andrews, Scotland
-#Y    Copyright (C) 2004, 2005, 2006 Marco Costantini
-##
-##  This file contains the GAP level interface to the output of the 
-##  lexical analyser gpipe.
-##  Comprises the "instream" functions - deals with stream at byte level
-##  and the "get" functions which get tokens and values from streams.
-##
-
-
-###########################################################################
-##
-##  Defining OpenMath tokens
```

Comment:
I am not so sure about this - for example, these tokens appeared in 2011 (https://github.com/gap-packages/openmath/commit/0abff1a89e3cb171b55c79b5e9d8e7d2994b0c0b) and may be related to Max Nicosia's work on supporting binary encoding for OpenMath. I will look during the pre-4.9.1 cleanup.